lib/config/recommended.js

```diff
@@ -1,4 +1,5 @@
 const rules = require('../recommended-rules');
+const util = require('ember-template-imports/src/util');
```

Comment:
(not-blocking)

I see there are several times we have to reach directly into [ember-template-imports](https://github.com/ember-template-imports/ember-template-imports) via file path to import something. The authors of that package may want to consider adding official exports for them so we can import them like this for example:

```js
const { TEMPLATE_TAG_PLACEHOLDER, preprocessEmbeddedTemplates } = require('ember-template-imports');
```

That would go nicely together with strictly-defining their node API via package.json `exports` which they might also decide to do at some point.

CC: @chriskrycho

Reply:
IMO, it's private API (or at the very least: intimate), so I don't think we actually want to expose it formally.